terraform/variables.tf

```diff
@@ -208,3 +208,10 @@ variable "restrict_kubeapi" {
   type        = list(string)
   default     = []
 }
+
+
+variable "capo_instance_create_timeout" {
+  description = "time to wait for an openstack machine to be created (in minutes)"
+  type        = number
+  default     = 10
```

Comment:
Is it wise to increase the default timeout so much?
Normally instance creation is well below 1min -- only if a port assignment fails and the VM needs to be rescheduled is when we see times in the 5 - 6min range.
I guess I would have chose 6mins as default allowing for 1 resched in an otherwise well-working cloud without making us wait for the error result too long.
(We will need longer timeouts once we spawn bare metal machines, but this is not the case yet, and I would probably then document this.)

Reply:
I used the same value the user did who filed the issue, since I didn't see any disadvantages. If there are, we probably should stay at 5 and let the user who needs more adjust it.

I don't think we will spawn bare metal machines using CAPO.